cmd/createtree/main.go

```diff
@@ -125,14 +128,15 @@ func newRequest(opts *createOpts) (*trillian.CreateTreeRequest, error) {
 	}
 
 	tree := &trillian.Tree{
-		TreeState:          trillian.TreeState(ts),
-		TreeType:           trillian.TreeType(tt),
-		HashStrategy:       trillian.HashStrategy(hs),
-		HashAlgorithm:      sigpb.DigitallySigned_HashAlgorithm(ha),
-		SignatureAlgorithm: sigpb.DigitallySigned_SignatureAlgorithm(sa),
-		DisplayName:        opts.displayName,
-		Description:        opts.description,
-		PrivateKey:         pk,
+		TreeState:             trillian.TreeState(ts),
+		TreeType:              trillian.TreeType(tt),
+		HashStrategy:          trillian.HashStrategy(hs),
+		HashAlgorithm:         sigpb.DigitallySigned_HashAlgorithm(ha),
+		SignatureAlgorithm:    sigpb.DigitallySigned_SignatureAlgorithm(sa),
+		DisplayName:           opts.displayName,
+		Description:           opts.description,
+		PrivateKey:            pk,
+		MaxRootDurationMillis: opts.maxRootDuration.Nanoseconds() / int64(time.Millisecond),
```

Comment:
What was the rationale?
`...Duration...` feels a little confusing to me, "is it how long the root is valid for, or ...", whereas `...Interval...` seems more likely to be understood as the maximum interval between roots - the help string for the flag starts "Interval after which...", same with the field documentation in the proto file...